app/queries/get_all_incidents_query.rb

```diff
@@ -8,7 +8,7 @@ def perform_query(params)
       if params[:user].admin?
         # Admins oversee incidents of all users in their ORI and contracting ORIs.
         params[:user].allowed_oris.flat_map do |ori|
-          User.where(ori: ori).all.flat_map do |user|
+          User.find_all_by_secondary_index(ori: ori).flat_map do |user|
```

Comment:
Nope! It'll scan by default, because `Dynamoid::Chain` doesn't know about secondary indices at the moment.